Notation.Plugin.AzureKeyVault/Notation.Plugin.AzureKeyVault.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>Notation.Plugin.AzureKeyVault</RootNamespace>
+    <RuntimeIdentifier>linux-x64</RuntimeIdentifier>
```

Comment:
Will this impact releasing other runtime bundles?

Reply:
This doesn't impact releasing other runtime bundles, however, I removed it.
Also removed:
```xml
<PublishSingleFile>true</PublishSingleFile>
    <PublishTrimmed>true</PublishTrimmed>
    <SelfContained>true</SelfContained>
```
So the config will be added as build flags in Release CI script.